deployment/base/deployment.yaml

```diff
@@ -8,6 +8,7 @@ spec:
     spec:
       containers:
       - name: sync-bridge
+        image: sensrnetnl/sync-bridge:latest
```

Comment:
Toch `latest`? Of is dat alleen vanuit build perspectief?

Reply:
Er zijn nieuwe versies van de componenten (v0.1) onderweg om te kunnen gaan deployen. Daar zitten de wijzigingen voor organizations in en de smart contracts. Bij die releases zal ik de templates updaten 😄